datastore/data-loader/requirements.txt

```diff
@@ -2,44 +2,54 @@
 # This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --no-emit-index-url ./requirements.in
+#    pip-compile --no-emit-index-url
```

Comment:
We keep getting pingpong with this. If you use `just pip-compile` you generate them all in a consistent way.
But maybe not something you want to deal with in this PR.